koku/koku/settings.py

```diff
@@ -416,7 +416,7 @@
 S3_BUCKET_NAME = ENVIRONMENT.get_value("S3_BUCKET_NAME", default="koku-reports")
 S3_BUCKET_PATH = ENVIRONMENT.get_value("S3_BUCKET_PATH", default="data_archive")
 S3_REGION = ENVIRONMENT.get_value("S3_REGION", default="us-east-1")
-S3_ENDPOINT = ENVIRONMENT.get_value("S3_ENDPOINT", default="s3.us-east-1.amazonaws.com")
+S3_ENDPOINT = ENVIRONMENT.get_value("S3_ENDPOINT", default="http://s3.us-east-1.amazonaws.com")
```

Comment:
Had to add the http prefix because the aws session `resource` parameter `endpoint_url` was giving me an invalid endpoint without it.  Will probably have to make the same change in e2e-deploy unless someone has a better solution.